deployment/ansible/roles/ashlar.app/templates/app-config.py.j2

```diff
@@ -0,0 +1,24 @@
+# Statement for enabling the development environment
```

Comment:
Add the typical "WARNING: This file auto-generated by ansible" comment header here